### PR TITLE
[JENKINS-65398] Terminology updates

### PR DESCRIPTION
**Pull request by Vlatombe**
Remove usage of master, slave, blacklist, whitelist

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by Vlatombe** [comment]
ping @ikedam

**Review by ikedam — APPROVED** [review]

**Comment by ikedam** [comment]
Changelog candidate (Reference: https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml#L3917)

```
=== 1.46.1

Release date: May xx, 2021

* Replaced text references to "slave" with "agent" in various log entries, help pages, and documentations. (https://issues.jenkins.io/browse/JENKINS-65398[JENKINS-65398])
```

File: src/main/resources/hudson/plugins/copyartifact/Messages_de.properties [hunk 1]
```diff
@@ -8,7 +8,7 @@ CopyArtifact.MissingProject=Konnte kein Projekt finden f\u00fcr die Artefaktkopi
 Das k\u00f6nnte an einem falschen Projektnamen oder fehlenden Zugriffsrechten liegen; \
 Vergleiche mit der Hilfe f\u00fcr Projektnamen in der Jobkonfiguration.
 CopyArtifact.MissingSrcArtifacts=Konnte nicht auf vorgelagerten Artefaktbereich zugreifen. Archiviert das Quellprojekt Artefakte?
-CopyArtifact.MissingSrcWorkspace=Konnte nicht auf vorgelagerten Arbeitsbereich f\u00fcr Artefaktkopie zugreifen. Slave-Knoten offline?
+CopyArtifact.MissingSrcWorkspace=Konnte nicht auf vorgelagerten Arbeitsbereich f\u00fcr Artefaktkopie zugreifen. Agent-Knoten offline?
```
Comment (Vlatombe) [inline src/main/resources/hudson/plugins/copyartifact/Messages_de.properties:11]:
```suggestion
CopyArtifact.MissingSrcWorkspace=Konnte nicht auf vorgelagerten Arbeitsbereich f\u00fcr Artefaktkopie zugreifen. Agenten-Knoten offline?
```

**Review by ikedam — APPROVED** [review]

**Comment by Vlatombe** [comment]
@ikedam do you need anything else on this PR?

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -630,8 +630,8 @@ Release date: Oct 10, 2010
 
 * Add option to copy from workspace of latest completed build.
 (https://issues.jenkins-ci.org/browse/JENKINS-7130[JENKINS-7130]) +
-Note: source and target jobs cannot both run on slave nodes; at least
-one must be on the master node.
+Note: source and target jobs cannot both run on agent nodes; at least
+one must be on the controller node.
```
Comment (daniel-beck) [inline CHANGELOG.adoc:634]:
```suggestion
one must be on the built-in node.
```

**Review by daniel-beck — CHANGES_REQUESTED** [review]

**Comment by ikedam** [comment]
@Vlatombe 
> do you need anything else on this PR?

The review looks still in progress to me.
Is there a deadline to make a new release with this fix?

File: src/main/webapp/help-workspace_de.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <div>
-  Wenn Quell- und Zielprojekt auf unterschiedlichen Slave-Knoten
+  Wenn Quell- und Zielprojekt auf unterschiedlichen Agent-Knoten
```
Comment (daniel-beck) [inline src/main/webapp/help-workspace_de.html:2]: Could we add a TODO in this file? It also still says `Master` for the built-in node, which does not have a German term yet.

**Review by daniel-beck — APPROVED** [review]

**Comment by daniel-beck** [comment]
Sorry, all reviews I leave in this org look like my opinion matters :-(

File: src/main/webapp/help-workspace_de.html (hunk 3 above)
Comment (Vlatombe) [reply src/main/webapp/help-workspace_de.html:3]: Done

**Comment by Vlatombe** [comment]
@ikedam Review looks good now. 

Sorry for being pushy with this, but we (CloudBees) try to make sure terminology updates are applied consistently across the Jenkins ecosystem, so we follow up on all open-source PRs related to that topic.

File: src/main/webapp/help-workspace_de.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
+<!-- TODO update master in german once a term is defined -->
 <div>
-  Wenn Quell- und Zielprojekt auf unterschiedlichen Slave-Knoten
+  Wenn Quell- und Zielprojekt auf unterschiedlichen Agent-Knoten
```
Comment (ikedam) [inline src/main/webapp/help-workspace_de.html:3]: I suppose this also should be `Agenten-Konten`.

Comment (ikedam) [reply src/main/webapp/help-workspace_de.html:3]:
```suggestion
  Wenn Quell- und Zielprojekt auf unterschiedlichen Agenten-Knoten
```

**Comment by ikedam** [comment]
I think I can make a new release tomorrow (might be today).
Thanks for the update!

**Comment by ikedam** [comment]
I released copyartifact-1.46.1. It will be available in the update center in a day.

**Pull request merged**